docs/pages/ktable.vue

```diff
@@ -58,9 +61,12 @@
       </DocsShow>
 
       <!-- Frontend Sorting Example-->
-      <h3>Table with Default Sorting</h3>
+      <h3>Table with Sorting</h3>
```

Comment:
Not introduced by this PR, just an opportunity for little improvement to help [the writing style](https://design-system.learningequality.org/writing/#punctuation) consistency

```suggestion
      <h3>Table with sorting</h3>
```